test/test_nn.py

```diff
@@ -4515,6 +4515,126 @@ def test_spectral_norm_pickle(self):
         m = pickle.loads(pickle.dumps(m))
         self.assertIsInstance(m, nn.Linear)
 
+    def test_orthogonal_parametrization(self):
+        # Orthogonal implements 6 algorithms (3x parametrizations times 2 options of extra_memory)
+
+        def assert_is_orthogonal(X):
```

Comment:
You can re-use the `_is_orthogonal` function here no?

Reply:
I wanted to use `assert_allclose` directly.

Also, this function transposes `X`. This would need to stay as `_is_orthogonal` expects `X` to be tall, not wide. All in all, I think it's not too bad to repeat those three lines.